core/src/main/java/hudson/Functions.java

```diff
@@ -1992,6 +1992,7 @@ public boolean hyperlinkMatchesCurrentPage(String href) {
         href = URLDecoder.decode(href, StandardCharsets.UTF_8);
         if (url.endsWith("/")) url = url.substring(0, url.length() - 1);
         if (href.endsWith("/")) href = href.substring(0, href.length() - 1);
+        if (href.startsWith("../")) href = href.substring(3, href.length());
```

Comment:
This seems kinda wrong. What is this for? Introduced in https://github.com/jenkinsci/jenkins/pull/7658/commits/ffac4a70843ec2445befd25543b5023f9c64c2fb but don't know what the message refers to.

Reply:
@timja ?

Reply:
It's the computer status link changed in https://github.com/jenkinsci/jenkins/pull/7658/files#diff-12cceca4e3d0a9025611e4ccea041aba4a53f5519f89beb8cd2a7dc6d05c9d41

(core/src/main/resources/hudson/model/ComputerSet/sidepanel.jelly)

because it was a relative link the existing `hyperlinkMatchesCurrentPage` function couldn't handle it and you had a link linking to the same page without clearly showing it, i.e. the link was supposed to be an active one, without this change it was broken.